scapy/contrib/cdp.py

```diff
@@ -371,7 +371,7 @@ def _check_len(self, pkt):
         This padding is only used for checksum computation.  The original
         packet should not be altered."""
         if len(pkt) % 2:
-            last_chr = pkt[-1]
+            last_chr = chb(pkt[-1])
```

Comment:
I'd rather use `orb()` here, since it will do nothing in Python 3 (on which I think we should focus now).
```suggestion
            last_chr = orb(pkt[-1])
```